sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -931,26 +913,18 @@ void setUncaughtExceptionHandler(UncaughtExceptionHandler handler) {
     }
 
     @VisibleForTesting
-    @Nullable
     ErrorReport buildErrorReport(ManagedErrorLog log) {
         UUID id = log.getId();
         if (mErrorReportCache.containsKey(id)) {
             ErrorReport report = mErrorReportCache.get(id).report;
             report.setDevice(log.getDevice());
             return report;
         } else {
-            File file = ErrorLogHelper.getStoredThrowableFile(id);
-            if (file != null) {
-                String stackTrace = null;
-                if (file.length() > 0) {
-                    stackTrace = FileManager.read(file);
-                }
-                ErrorReport report = ErrorLogHelper.getErrorReportFromErrorLog(log, stackTrace);
-                mErrorReportCache.put(id, new ErrorLogReport(log, report));
-                return report;
-            }
+            String stackTrace = log.getException() == null ? null : log.getException().getStackTrace();
```

Comment:
It looks like you didn't support data from old versions of SDK. You should keep reading as backup for now